ssh-server/Dockerfile

```diff
@@ -1,13 +1,10 @@
-FROM debian:buster-slim
+FROM debian:bookworm-slim
 
 RUN apt-get update
 RUN apt-get install -y autoconf automake gcc git gzip libssl-dev libz-dev make openssl
 
 RUN cd /tmp; \
-    git clone https://github.com/mkj/dropbear; \
+    git clone https://github.com/mkj/dropbear -b DROPBEAR_2024.85; \
```

Comment:
nit: This is now pinning the version. Is this on purpose / necessary?

Reply:
Yes, I specified the tag to indicate that a stable release was used, and should help associate version to the builds in the future.

The dropbear binaries are built manually and uploaded i.e. there's no CI automation, so there's no particular advantage to not pinning.